tools/payments/report.go

```diff
@@ -235,6 +235,10 @@ func (r PreparedReport) Prepare(ctx context.Context, key ed25519.PrivateKey, cli
 
 	reqs := make([]payments.PrepareRequest, len(r))
 	for i, paymentDetails := range r {
+		// Implement basic maximum payout amount protection
+		if paymentDetails.Amount.GreaterThan(decimal.NewFromInt32(50)) {
```

Comment:
Nit: the limit could be moved up outside of the loop.

Reply:
Realized we should reuse the value defined in `clients` until we make it configurable.